daisy_workflows/image_import/windows/translate_bootstrap.ps1

```diff
@@ -160,8 +160,8 @@ try {
   }
 
   $driver_dir = 'c:\drivers'
-  New-Item $driver_dir -Type Directory | Out-Null
-  New-Item $script:components_dir -Type Directory | Out-Null
+  New-Item $driver_dir -Force -Type Directory | Out-Null
```

Comment:
Force creation of this directory *shouldn't* be necessary, was there an error for a particular image import?

Reply:
I failed to import a image that is exported from GCE. So I added -Force for all folder creation. I think it's harmless, is it?

Reply:
If the intent of this PR is to allow for multiple re-imports/exports of an image on GCP, this -Force parameter is NOT necessary, and should be removed.